### PR TITLE
Add package morse2ascii

### PR DESCRIPTION
**Pull request by Auxilus**

### CODE REVIEW

File: packages/morse2ascii/Makefile.patch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--- ./Makefile
++++ ../Makefile
+@@ -1,6 +1,6 @@
+ EXE		= morse2ascii
+ CFLAGS	+= -O2 -s
+-PREFIX	= /usr/local
++PREFIX	= /data/data/com.termux/files/usr
```
Comment (stephengroat) [inline packages/morse2ascii/Makefile.patch]: Might be better to set `PREFIX = $TERMUX_PREFIX`?

File: packages/morse2ascii/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://packages.debian.org/sid/morse2ascii
+TERMUX_PKG_DESCRIPTION="A tool  to decode the morse codes from a PCM WAV file as well as from text and RAW PCM files"
```
Comment (stephengroat) [inline packages/morse2ascii/build.sh]: extra space

File: packages/morse2ascii/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+TERMUX_PKG_VERSION=0.2
+#TERMUX_PKG_REVISION=2
+TERMUX_PKG_SRCURL=http://ftp.debian.org/debian/pool/main/m/morse2ascii/morse2ascii_${TERMUX_PKG_VERSION}+dfsg.orig.tar.xz
+TERMUX_PKG_SHA256=0d41f544818f27db2fb211b2950e48893ff4856f1d114ab1878c91e6e27e4c49
+#TERMUX_PKG_DEPENDS="libconfuse, libnl, ncurses"
```
Comment (stephengroat) [inline packages/morse2ascii/build.sh]: can delete commented lines

File: packages/morse2ascii/Makefile.patch (hunk 1 above)
Comment (Grimler91) [reply packages/morse2ascii/Makefile.patch]: Or even @TERMUX_PREFIX@ as in other patches :+1:

Comment (stephengroat) [reply packages/morse2ascii/Makefile.patch]: or just take @Grimler91 which is MUCH better

**Comment by Auxilus** [comment]
@stephengroat @Grimler91 thanks for reviewing!

**Comment by Wetitpig** [comment]
So now are we going too add this package?

**Comment by xeffyr** [comment]
Implemented in [unstable-packages](https://github.com/termux/unstable-packages) repository.
```
pkg install unstable-repo
pkg install morse2ascii
```

**Pull request closed without merging**